compiler/src/dmd/mtype.d

```diff
@@ -4893,6 +4990,17 @@ extern (C++) final class TypeFunction : TypeNext
     {
         v.visit(this);
     }
+
+    /// Returns: index of parameter with name `ident` or -1 if not found
+    private extern(D) ptrdiff_t parameterIndex(Identifier ident)
```

Comment:
Search functions should be prefixed with `find`, as in `findParameterIndex`. Also, use Ddoc `params:` section

Reply:
I'll rename it